src/domains/pay-wallet-common/03_cosmosdb.tf

```diff
@@ -136,6 +143,20 @@ locals {
       ]
       shard_key = "walletId"
     },
+    { # collection with new and detailed logging events
+      name                = "payment-wallets-log-events"
+      default_ttl_seconds = "2592000" #30 days
```

Comment:
why 30 days? if we estimate how many events we can have per month maybe we can extend to 90 days. So larger time window for support.

Reply:
Considering that the current events collection, that has been created about 2 weeks ago, contains about 560k documents, we could estimate roughly 1.2M events per month; could we extend the TTL to 90 days given this estimation?

Reply:
I propose at least 60 days which is enough time to recover any events for debugging,
"ortofruttifero" poll? what you think? @pagopa/pagopa-team-touchpoint

Reply:
🍎